src/Microsoft.AspNetCore.Mvc.Core/Controllers/DefaultControllerFactory.cs

```diff
@@ -44,17 +39,17 @@ IEnumerable<IControllerPropertyActivator> propertyActivators
                 throw new ArgumentNullException(nameof(propertyActivators));
             }
 
-            _controllerActivator = controllerActivator;
+            ControllerActivator = controllerActivator;
             _propertyActivators = propertyActivators.ToArray();
         }
 
         /// <summary>
         /// The <see cref="IControllerActivator"/> used to create a controller.
         /// </summary>
-        protected IControllerActivator ControllerActivator => _controllerActivator;
+        private IControllerActivator ControllerActivator { get; }
```

Comment:
Is there any reason to move these things into props and make them explicitly private? I would have left the private readonly var instead. We don't do private properties anywhere AFAIK